SPECS/perftest/perftest.spec

```diff
@@ -0,0 +1,80 @@
+Summary:        IB Performance tests
+Name:           perftest
+# Update extended_release with version updates
+Version:        4.5
+Release:        1%{?dist}
+License:        BSD 3-Clause, GPLv2
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Productivity/Networking/Diagnostic
+URL:            https://www.openfabrics.org
+Source0:        https://linux.mellanox.com/public/repo/doca/1.3.0/extras/mlnx_ofed/5.6-1.0.3.3/SOURCES/perftest_4.5.orig.tar.gz#/%{name}-%{version}.tar.gz
```

Comment:
Considering the `5.6-1.0.3.3` part, should this package somehow always follow the OFED scripts one in terms of versions? If yes, we should at least add a comment.